.github/release-please.yml

```diff
@@ -1 +1,2 @@
 releaseType: node
+packageName: '@google-cloud/storage'
```

Comment:
Grump.  Will we need to specify this anywhere?  Any reason we can't just read this from `package.json`?  All of the duplicate config makes me nervous.

Reply:
I thought about reading it from `package.json`, but it would mean that `release-please` needs to read in two config files, which is silly, and not every language has a `package.json`.

I think this is reasonable because it's a config setting specifically for `release-please`, and this is a config file for release-please.